### PR TITLE
Dry up client

### PR DESCRIPTION
**Pull request by silasdavis**
wet dog

### CODE REVIEW

File: rpc/tendermint/client/client.go [hunk 1]
```diff
@@ -159,36 +94,68 @@ func CallCode(client rpcclient.Client,
...
+		_, err = cli.Call(method, paramsSlice, &res)
+	case *rpcclient.ClientURI:
+		_, err = cli.Call(method, paramsMap, &res)
+	}
+	return res, err
+
+}
+func mapAndValues(orderedKeyVals ...interface{}) (map[string]interface{}, []interface{}) {
+	if len(orderedKeyVals)%2 != 0 {
+		panic(fmt.Sprintf("mapAndValues requires a even length list of keys and "+
```
Comment (benjaminbollen) [inline rpc/tendermint/client/client.go]: These panics should not be panics, rather errors.  I can see why you use a panic here; but I would only want to move back to panics here, if we had asserted coverage of all calls to `mapAndValues` in the CI; right now this relies on our continued vigilance.

Comment (silasdavis) [inline rpc/tendermint/client/client.go]: Yeah I guess I could have just let it panic array out of bounds, so I saw this as making the panic nicer. But I agree.

**Comment by silasdavis** [comment]
@benjaminbollen Stopped panicking

**Comment by benjaminbollen** [comment]
LGTM

**Pull request merged**